implementations/rust/ockam/ockam_api/src/control_api/openapi.rs

```diff
@@ -24,12 +26,93 @@ impl Modify for Authentications {
     }
 }
 
+const MAIN_DESCRIPTION: &str = r#"
+## Overview
+This API is designed to control Ockam nodes via HTTP requests without having to use Ockam Command or Ockam library.
+You can expect a similar abstraction level as Ockam Command, but with a more programmatic approach.
+
+The endpoint will receive the HTTP request and forward it to the selected node using the `node`
+parameter. How the node will be selected starting from the name is configuration dependent.
```

Comment:
How can the user find out more about this? Do that really concern them?

Reply:
We haven't defined yet how to start the service 😅 ,  so i was hard-pressed to add more information.
Once we deploy, definitely I'll need to make some adjustments to the API and the documentation